users/README.md

```diff
@@ -34,6 +34,11 @@
 		"super_user": "boolean",
 		"in_discord": "boolean"
 	},
+        "archivedDetails": [{
+                "updated_at": "timestamp",
+                "super_user_id": "string",
+                "reason": "string"
+         }],
```

Comment:
Do you know if we need to store the archived details in the user collection?

Should this be in logs or have a separate collection?

Reply:
what do you mean by logs?
I'm not sure about where to store archive details. Is not it supposed to be in the user's collection?